spec/game_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'date'
+require_relative 'spec_helper'
+
+describe Game do
+  before :each do
+    @game = Game.new(true, '2022-10-11', '2019-11-11')
+  end
+
+  describe '#new' do
+    it 'takes three parameters and return game object' do
+      expect(@game).to be_an_instance_of Game
+    end
+  end
+
+  describe '#multiplayer' do
+    it 'return the true for multiplayer' do
+      @game.multiplayer.should eql true
+    end
+  end
+
+  describe '#last_played_at' do
+    it 'return the true for last_played_at' do
+      @game.last_played_at.should eql Date.parse('2022-10-11')
+    end
```

Comment:
- [ ] Please kindly check the description of your `it` block, What you are expecting to be is not `true` but `date` so you can modify your description